rclc_parameter/test/rclc_parameter/test_parameter.cpp

```diff
@@ -475,31 +485,45 @@ TEST_P(ParameterTestBase, rclc_delete_parameter) {
 
   // Fail on deleted parameter
   EXPECT_EQ(rclc_delete_parameter(&param_server, param_name), RCL_RET_ERROR);
+
+  // No callback calls
+  ASSERT_EQ(callback_calls, 0U);
 }
 
 TEST_P(ParameterTestBase, rclcpp_delete_parameter) {
-  int expected_callcack_calls = 1;
+  size_t expected_callcack_calls = 1;
```

Comment:
```suggestion
  size_t expected_callback_calls = 1;
```
Opss, name error repeated everywhere.